dkc/core/rest/file.py

```diff
@@ -115,6 +117,18 @@ def perform_create(self, serializer: FileSerializer):
                 {'size': ['This file would exceed the size quota for this folder.']}
             )
 
+    def perform_update(self, serializer: FileUpdateSerializer) -> None:
+        with transaction.atomic():
+            # We lock this file row in order to make sure blob can only be set once
```

Comment:
This is the what, can you elaborate to provide the why in the comment, as the intent isn't clear to me from reading this?